### DISCUSSION TITLE
Symfony version update issue

### DISCUSSION THREAD

**Discussion by gopikrish19** [symfony/symfony#51054] [Q&A]
Hi,

 I have updated my symfony application from 6.2.9 to 6.3.1, once updated the version, I am getting the error from doctrine orm mapping 

 error: **Warning: Attempt to read property "name" on string**

Once i downgrade the version previous version, its working fine, Please let me know the issue with this update

**Comment by MatTheCat** [comment]
Hello, could you provide a stack trace or [a reproducer](https://symfony.com/doc/current/contributing/code/reproducer.html)?

**Reply by gopikrish19** [reply]
Please check this

in [/var/www/html/symfony-traggtor.agry.app/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/Driver/AttributeDriver.php ](line 694)

**Reply by MatTheCat** [reply]
This is neither a stack trace nor a reproducer.

And it has nothing to do with Symfony either :sweat_smile: look at https://github.com/doctrine/orm/issues/10690 your mapping may be wrong.
